### DISCUSSION TITLE
Can ord apply compression to audio, video, and image inscriptions?

### DISCUSSION THREAD

**Discussion by jolow99** [ordinals/ord#1999] [Q&A]
could I ask if there are any CLI command flags I can include to specify enable / disable compression? does the ord utility itself apply compression to audio?

**Comment by soenkehahn** [comment]
No, the `ord` program does not modify the files that you give to the `ord wallet inscribe` command in any way. This is intentional to give users full control over the exact contents of the inscriptions. If you want to inscribe compressed files, you have to compress them yourself before inscribing.
